pkg/controller/nstemplateset/namespaces_test.go

```diff
@@ -324,8 +323,8 @@ func TestEnsureNamespacesOK(t *testing.T) {
 
 	t.Run("should create the second namespace when the first one already exists", func(t *testing.T) {
 		// given
-		nsTmplSet := newNSTmplSet(namespaceName, username, "basic", withNamespaces("dev", "code"), withConditions(Provisioning()))
-		devNS := newNamespace("basic", username, "dev", withTemplateRef())
+		nsTmplSet := newNSTmplSet(namespaceName, username, "basic", withNamespaces("abcde11", "dev", "code"), withConditions(Provisioning()))
+		devNS := newNamespace("basic", username, "dev", withTempalteRefUsingRevision("abcde11"))
```

Comment:
there's a small typo in the func name: `withTempalteRefUsingRevision` -> `withTemplateRefUsingRevision`

Reply:
fixed in 7ff9a06
Thanks :+1: